swig/cpp/tests/changes.cpp

```diff
@@ -0,0 +1,219 @@
+#include <iostream>
+#include <memory>
+#include <cassert>
+#include <cstring>
+#include <unistd.h>
+
+#include "Session.h"
+
+#define MAX_LEN 100
+
+using namespace std;
+
+const char *module_name = "swig-test";
+const int LOW_BOUND = 10;
+const int HIGH_BOUND = 20;
+const char *xpath_if_fmt = "/swig-test:cpp-operations/test-get[name='%s']/%s";
+
+char *
+get_test_name(int i)
+{
+    char buf[MAX_LEN] = {0};
+    sprintf(buf, "test-cpp-%d", i);
+    return strdup(buf);
+}
+
+const char *
+get_xpath(char *test_name, char *node_name)
```

Comment:
I've done it like that because wrapper functions return C types. If std::string is preferable in  C++ then we'll change it like that. Do you think bindings should be updated to return std::string instead of char* in general?